app/assets/stylesheets/components/_icon.scss

```
@@ -36,3 +36,28 @@ $icon-min-padding: 2px;
     }
   }
 }
+
+// Login Specific Icons (non-uswds)
+.lg-icon {
+  width: 88px;
+  height: 88px;
+}
+
+// For displaying icons as "badges"
+// at the top of modals
+.iconic-modal-badge {
+  &::before {
+    background-repeat: no-repeat;
+    background-size: contain;
+    content: '';
+    position: absolute;
+    left: calc(50% - 44px);
+    top: 0px;
+    transform: translateY(-50%);
```

Minor: The X and Y positioning seem like we're using two approaches to do the same thing. Should we pick one?

```suggestion
    left: calc(50% - 44px);
    top: -44px;
```

```suggestion
    left: 50%;
    top: 0px;
    transform: translateX(-50%) translateY(-50%);
```